mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -4,12 +4,15 @@
 import time
 from tqdm import tqdm
 import sys
+from db.insert_data import insert_into_mongo
+from constants import *
 
 class WikidataServiceTimeoutException(Exception):
     pass
 
-data_dir = os.path.join(os.path.dirname(__file__), '../../data')
-out_dir = os.path.join(os.path.dirname(__file__), '../../out')
+# data_dir = os.path.join(os.path.dirname(__file__), '../../data')
```

Comment:
nit: no need to leave commented code, unless it will be used in a future PR.

Reply:
Have removed them